backtrace-library/src/main/java/backtraceio/library/common/RequestHelper.java

```diff
@@ -0,0 +1,53 @@
+package backtraceio.library.common;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import backtraceio.library.logger.BacktraceLogger;
+
+public class RequestHelper {
+    private static final transient String LOG_TAG = RequestHelper.class.getSimpleName();
+
+    private static final String CRLF = "\r\n";
+
+    /**
+     * Get Content-Type of request
+     *
+     * @return string with content type and information about boundary
+     */
+    public static String getContentType() {
+        return "application/json";
```

Comment:
Us existing constant in library?

Reply:
it's a small nit - without it we should accept pull request - not_ sure if this will add what Vincent wants, but I see MediaType.APPLICATION_JSON gives you what you want.

Reply:
I can try using that constant. I just followed similar pattern as @BartoszLitwiniuk code:
https://github.com/backtrace-labs/backtrace-android/blob/dcf0eb97675d693c5aff0f733d407197e9dad168/backtrace-library/src/main/java/backtraceio/library/common/MultiFormRequestHelper.java#L27

Reply:
Correct me if I'm wrong but in Android SDK there is not const/enum for `multipart/form-data;boundary=`

Reply:
[MediaType.APPLICATION_JSON is only available in javax](https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/MediaType.html). [Javax is not always available in Android](https://stackoverflow.com/a/16811778). I don't think there is any existing constant for this so I will mark this as resolved @vlussenburg @konraddysput